torch/utils/data/dataloader.py

```diff
@@ -233,6 +237,16 @@ def __init__(self, dataset, batch_size=1, shuffle=False, sampler=None,
         self.__initialized = True
         self._IterableDataset_len_called = None  # See NOTE [ IterableDataset and __len__ ]
 
+        self._persistent_workers = persistent_workers
+        self._iterator = self._get_iterator() if persistent_workers else None
```

Comment:
is it intended that the workers are started when `DataLoader` is created (rather than when the first iterator of `DataLoader` is created)?